components/brave_sync/sources.gni

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+brave_components_history_core_browser_deps =
```

Comment:
actually this doesn't look right. this file either should be located in `components/history/core/browser/sources.gni`, or `components/sync` should declare `//brave/components/brave_sync:features` in `public_deps`.

Reply:
Thanks, fixed at 640f6183d634824a57336b4921758e3010364f94 .